openedx/core/djangoapps/user_api/management/tests/test_cancel_retirement.py

```diff
@@ -0,0 +1,75 @@
+"""
+Test the cancel_user_retirement_request management command
+"""
+import pytest
+
+from django.contrib.auth.models import User
+from django.core.management import call_command, CommandError
+from openedx.core.djangoapps.user_api.models import UserRetirementStatus, RetirementState
+from student.tests.factories import UserFactory
+from student.models import get_retired_email_by_email
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def retirement_user():
+    return UserFactory.create(username='test_user')
+
+
+def _create_retirement_request(user):
```

Comment:
It's a bummer that the helper functions which do these things in `retirement_helpers.py` are all class based. Maybe worth breaking them out to functions and using those in the class instead?